client/src/components/Profile/UserInfo.js

```diff
@@ -32,8 +33,11 @@ const UserInfo = props => {
     const fetchData = () => {
         axios.get('profile/60021edaab2f25167778e7f9').then(res => {
             console.log(res.data);
-            console.log(res.data.firstName);
             setFirstName(res.data.firstName);
+            setLastName(res.data.lastName);
+            setEmail(res.data.user.email);
+            setLocation(res.data.location);
+            setPhone(res.data.phone);
         });
     };
```

Comment:
I'm trying to use setFirstName but it won't work in the defaultValue for the TextField components.